src/audio/eq_fir/eq_fir_ipc3.c

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2017 Intel Corporation. All rights reserved.
+//
+// Author: Seppo Ingalsuo <seppo.ingalsuo@linux.intel.com>
+//         Liam Girdwood <liam.r.girdwood@linux.intel.com>
+//         Keyon Jie <yang.jie@linux.intel.com>
+
+#include "eq_fir.h"
```

Comment:
no, you have to include all the headers, that this file uses here explicitly. We don't use gather-all headers

Reply:
I think you're still missing some headers (see below)